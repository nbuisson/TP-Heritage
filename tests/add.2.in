R 1 2 3 
R 1 Eaz 1
C 1 2 -1
EXIT
