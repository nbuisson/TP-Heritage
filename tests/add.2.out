ERR R 1 2 3
ERR R 1 Eaz 1 
ERR C 1 2 -1 
0
